packages/block-library/src/image/save.js

```diff
@@ -27,6 +27,9 @@ export default function save( { attributes } ) {
 	} = attributes;
 
 	const newRel = isEmpty( rel ) ? undefined : rel;
+	if ( ! url )  {
```

Comment:
```suggestion
	if ( ! url ) {
```

There are two spaces, we need to remove one. We also can move it above the line with `const newRel = ...`.